socorro/external/postgresql/new_crash_source.py

```diff
@@ -1,146 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The crash source is what tells the processor which crashes need to be processed. We're using the RabbitMQ crash source. We can nix all the other ones including this one.

Figured I'd do that while I was removing things.